confdgnmi/testtool/OpenConfig/openconfig.resource

```diff
@@ -12,12 +13,17 @@ Verify Get of
     When Dispatch Get Request
     Then Should Received Ok Response
 
-Verify Get of "{path}" with supported encodings
-    Get Capabilities From Device
-    ${supported_encodings}=  Last Supported Encodings
-    FOR  ${encoding}  IN  @{supported_encodings}
-        Log    ${encoding}
-    END
+Iterate Get of
+    [Arguments]  ${path}
+    Verify Get of  ${path}
+    [Teardown]    Teardown gNMI state
+
+# Verify Get of "{path}" with supported encodings
```

Comment:
Is this keyword unused? Then it should be removed, shouldn't it?

Reply:
correct, will remove...